spec/screens/creator_spec.rb

```diff
@@ -13,31 +13,31 @@
   end
 
   it "saves the name introduced by the user" do
-    allow(ui).to receive(:ask_for_name).and_return("name")
+    allow(ui).to receive(:ask_for_value_to_edit).with("name").and_return("name")
     creator.run
     expect_field("name", "name")
   end
 
   it "saves the address introduced by the user" do
-    allow(ui).to receive(:ask_for_address).and_return("address")
+    allow(ui).to receive(:ask_for_value_to_edit).with("address").and_return("address")
     creator.run
     expect_field("address", "address")
   end
 
   it "saves the phone introduced by the user" do
-    allow(ui).to receive(:ask_for_phone).and_return("123456")
+    allow(ui).to receive(:ask_for_value_to_edit).with("phone").and_return("123456")
     creator.run
     expect_field("phone", "123456")
   end
 
   it "saves the email introduced by the user" do
-    allow(ui).to receive(:ask_for_email).and_return("email@mail.com")
+    allow(ui).to receive(:ask_for_value_to_edit).with("email").and_return("email@mail.com")
     creator.run
     expect_field("email", "email@mail.com")
   end
 
   it "saves the notes introduced by the user" do
-    allow(ui).to receive(:ask_for_notes).and_return("notes")
+    allow(ui).to receive(:ask_for_value_to_edit).with("notes").and_return("notes")
```

Comment:
Do you think it would make all the tests more readable if you would have a hand rolled `UI` and `DB` double?
There are _a lot_ `allow` calls in the screen tests (finder, creator, etc.).